src/sentry/tasks/relay.py

```diff
@@ -46,9 +48,10 @@ def update_config_cache(
     sentry_sdk.set_tag("update_reason", update_reason)
     sentry_sdk.set_tag("generate", generate)
 
-    # Not running this at the beginning of the task to add tags in case there's
-    # something wrong going on.
-    if not should_update_cache(
+    # Weird way to experiment with this easily.
```

Comment:
If this is meant to be removed soon, please leave a `TODO` here, especially since you're mentioning "weird".